modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/portlet/action/ActionUtil.java

```diff
@@ -67,6 +69,13 @@ public static MBCategory getCategory(HttpServletRequest httpServletRequest)
 
 		if (categoryId > 0) {
 			category = MBCategoryServiceUtil.getCategory(categoryId);
+
+			if (category.getGroupId() != themeDisplay.getScopeGroupId()) {
```

Comment:
Question:

Categories of ancestors (global, parent) sites should be allowed?

Reply:
With this approach, no, but it makes a lot of sense.

Reply:
so, ask change? or "patápalante"

Reply:
I'll contact you in slack to analyze this together (in a non public place)

Reply:
With this change the portlet will honor it's own scope configuration, so I think this is the right behavior.

Reply:
<img src="https://media4.giphy.com/media/R8RF7c7vH7Y8D5CYdP/giphy.gif"/>